tests/unit/storages/test_key_value_store.py

```diff
@@ -100,3 +100,12 @@ async def test_static_get_set_value(key_value_store: KeyValueStore) -> None:
     await key_value_store.set_value('test-static', 'static')
     value = await key_value_store.get_value('test-static')
     assert value == 'static'
+
+
+async def test_static_get_public_url(key_value_store: KeyValueStore) -> None:
+    await key_value_store.set_value('test-static', 'static')
+    public_url = key_value_store.get_public_url('test-static')
+    assert (
+        public_url
+        == f'file://{key_value_store._configuration.storage_dir}/storage/key_value_stores/default/test-static'
+    )
```

Comment:
This is not very useful. Could you perhaps try to open and read the URL instead and check if the content is actually `static`?

Reply:
Since kv is created by a generator, it yields a different value each time. Can you please help me with how I can make that static?

Reply:
I'm not sure I understand. Why can't you just do this?

```py
from urllib.parse import urlparse, unquote
from pathlib import Path
# ...

path = unquote(urlparse(public_url).path)
assert Path(path).read_text() == 'static'
```

Reply:
Sorry I mixed up the value static and path being static as in constant. I have updated the test case now